recipe/meta.yaml

```diff
@@ -71,6 +71,7 @@ outputs:
         - {{ compiler('cxx') }}
         - bazel <=3.4.1  # [not win]
         - bazel <3.7     # [win]
+        - sysroot_linux-64 2.17  # [linux64]
```

Comment:
if we update the base to `cos7` do we need to manually add this?..

Reply:
It's necessary so that this package gets the correct run-export (so that packages depending on ray fail to build if they don't have cos7 themselves), otherwise we might get crashes with the ABI.